documentation/release_notes.rst

```diff
@@ -11,14 +11,74 @@ creating efficient heterogeneous applications.
+release. Please use the relevant compiler flags to enable this feature.
+
+New Features
+------------
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``set_intersection``, ``set_union``,
+  ``set_difference``, ``set_symmetric_difference``, ``includes``, ``unique``, ``unique_copy``, ``destroy``,
+  ``uninitialized_fill``, ``uninitialized_move``, ``uninitialized_copy``, ``uninitialized_value_construct``,
+  ``uninitialized_default_construct``, ``reverse``, ``reverse_copy``, ``swap_ranges``. These algorithms operate with
+  C++20 random access ranges.
+- Improved performance of ``oneapi::dpl::experimental::kt::gpu::inclusive_scan`` and added support for binary operator
+  and type combinations which do not have a SYCL known identity.
+- Improved performance of ``inclusive_scan_by_segment`` and ``exclusive_scan_by_segment`` with
+  device execution policies on GPUs.
+- Improved performance of set operation algorithms when using device policies: ``set_union``, ``set_difference``,
+  ``set_intersection``, ``set_symmetric_difference``.
+- Improved performance of search operations (e.g., ``find``, ``all_of``, ``equal``, ``search``, etc.), ``is_heap`` and
+  ``is_heap_until`` algorithms on Intel® Arc™ B-series GPU devices.
+- Removed requirement of GPU double support to use ``set_union``, ``set_difference``, ``set_intersection``, and
+  ``set_symmetric_difference`` on Windows operating systems.
+- Removed copy constructible requirements from value type for ``reduce`` and ``transform_reduce`` algorithms when used
+  with a host policy, and default constructible requirements from value type for ``reduce`` and ``transform_reduce``
+  when used with any execution policy.
+
+Fixed Issues
+------------
+- Fixed an issue with incorrect handling of different projections by range-based ``merge``.
+- Fixed ``equal`` returning a ``false`` for empty input sequences; now it returns ``true``.
+- Fixed a compilation error **SYCL kernel cannot use exceptions** occurring with libstdc++ version 10 when calling
+  range-based ``adjacent_find``, ``is_sorted`` and ``is_sorted_until`` algorithms using  device policies.
+- Fixed an issue with ``PSTL_USE_NONTEMPORAL_STORES`` macro having no effect.
+- Fixed a bug where ``unique`` when called with a device execution policy returned an incorrect result
+  iterator.
+- Fixed a bug in ``exclusive_scan``, ``inclusive_scan``, ``transform_exclusive_scan``, ``transform_inclusive_scan``,
+  ``exlusive_scan_by_segment``, and ``inclusive_scan_by_segment`` algorithms when using device policies with differing
+  ``InputIterator`` and ``OutputIterator`` types.
+- Fixed error in return value types for oneapi::dpl::ranges::minmax_element and oneapi::dpl::ranges::mismatch algorithms.
+- Fixed compile errors in ``set_union`` and ``set_symmetric_difference`` for hetero execution policies when the types of
```

Comment:
I think instead of hetero we've used the term device execution policies more commonly.
```suggestion
- Fixed compile errors in ``set_union`` and ``set_symmetric_difference`` with device execution policies when the types of
```